config/services.yaml

```diff
@@ -21,6 +21,10 @@ services:
         public: true
         autoconfigure: false
 
+    App\Controller\Admin\:
+        resource: '../src/Controller/Admin'
+        public: true
```

Comment:
Recreated a route config because Compotes' controllers are all currently using `autoconfigure: false`, whereas EasyAdmin ones all need `autoconfigure: true`, because, well, FrameworkBundle's AbstractController 😑